pkg/virt-api/webhooks/validating-webhook/admitters/migrationpolicy-admitter.go

```diff
@@ -91,6 +97,21 @@ func (admitter *MigrationPolicyAdmitter) Admit(ar *admissionv1.AdmissionReview)
 		}
 	}
 
+	if spec.AllowPostCopy != nil && *spec.AllowPostCopy {
+		namespace, err := admitter.Client.CoreV1().Namespaces().Get(context.Background(), policy.Namespace, metav1.GetOptions{})
+		if err != nil {
+			return webhookutils.ToAdmissionResponseError(err)
+		}
+
+		if !psa.IsNamespacePrivileged(namespace) && !admitter.ClusterConfig.PSASeccompAllowsUserfaultfd() {
```

Comment:
nit: We could check the feature gate first and therefore we could avoid the API call in most cases.